### DISCUSSION TITLE
fast_Herbie_xarray MergeError

### DISCUSSION THREAD

**Discussion by v7nguyen** [blaylockbk/Herbie#76] [General]
Hi Brian,

When attempting to get the HRRR forecast for the most recent datetime for 48 hours (right now it is "2022-05-16 06:00") it is returning the following error: MergeError: conflicting values for variable 't2m' on objects to be combined. You can skip this check by specifying compat='override'. 

I attempted to pass in compat='override' as an argument in fast_Herbie_xarray but that did not work. When pulling the forecast for "2022-05-16 00:00", I do not get the following error. Not entirely sure what the issue is. 

Let me know if you are able to replicate this by pulling the most recent 48 hour forecast (every 6 hours UTC time).

![image](https://user-images.githubusercontent.com/53784737/168606651-b5ee6e48-362e-47fd-b81e-a0666b4a03e7.png)
![image](https://user-images.githubusercontent.com/53784737/168606779-8d95f856-6a47-434a-ab9c-cdb6b10e53ff.png)
![image](https://user-images.githubusercontent.com/53784737/168606898-113ab809-9eef-41bb-9f1a-d415dba497a5.png)
![image](https://user-images.githubusercontent.com/53784737/168606945-81ddefae-1de3-4833-8cec-5adce8f1d6a1.png)

**Comment by v7nguyen** [comment]
Interestingly, when I delete the HRRR data in the data folder and redownload an hour later, it works without errors. I am wondering if some files were incomplete when I was first downloading the files.

**Reply by blaylockbk** [reply]
That is the most likely possibility. The merge error problem is with xarray.concat when the datasets you're merging are not all similar. It could be that one of the files was missing 2 m temperature when the others had it.
